keyboards/whitefox/matrix.c

```diff
@@ -69,7 +69,7 @@ uint8_t matrix_scan(void)
             case 8: palSetPad(GPIOC, 11);   break;
         }
 
-        wait_us(1); // need wait to settle pin state
+        wait_us(10); // need wait to settle pin state
```

Comment:
I suggest that you change this to 20 us, like it's on the Ergodox. It probably works with 10us right now, but if you spawn other threads, it might stop working. 20us is still very fast, as you can do a full scan in 9 * 20us = 180us. For a comparison, the Ergodox Ez takets 3.15 ms, so it's still 7.5 times faster than that.

I don't know the exact reason why the delay has to be at minimum two ticks, but I suspsect it's due to how the context switching is implemented internally in ChibiOS.